src/ops/module.rs

```diff
@@ -40,7 +40,7 @@ pub enum ModuleOp {
         definition: SimpleType,
     },
     // A constant value definition.
-    Const(ConstValue),
```

Comment:
since you can always get the type from the value, seems no need to store it here?